### PR TITLE
IOR with foss2016a

### PR DESCRIPTION
**Pull request by RvDijk**

### CODE REVIEW

File: easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+source_urls = [('https://github.com/chaos/ior/archive/')]
+sources = ['%(version)s.tar.gz']
+
+# gmake posix      -- IOR with only POSIX interface
+# gmake mpiio      -- IOR with only POSIX and MPIIO interfaces
+# gmake hdf5       -- IOR with POSIX, MPIIO, and HDF5 interfaces
+# gmake ncmpi      -- IOR with POSIX, MPIIO, and NCMPI interfaces
+# gmake all        -- IOR with POSIX, MPIIO, HDF5, and NCMPI interfaces
+preconfigopts = "./bootstrap &&" 
+#buildopts = ' mpiio'
```
Comment (boegel) [inline easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a.eb]: @RvDijk did the build procedure change?

If so, please just drop this line, and the comments outlining the options above?

File: easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a.eb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+files_to_copy = [(['src/C/IOR'], 'bin'), 
+                "README", "RELEASE_LOG", "UNDOCUMENTED_OPTIONS", "USER_GUIDE", "scripts", "testing"]
```
Comment (boegel) [inline easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a.eb]: drop this, no longer used if you're not using `MakeCp` anymore?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2518.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a4cf5f26e983a4bb8bbd34595210e0df for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2145.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b1895f1766d4ff78cee6f4f8d5bc1864 for a full test report.

File: easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a-mpiio.eb [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+toolchain = {'name': 'foss', 'version': '2016a' }
+
+source_urls = [('https://github.com/chaos/ior/archive/')]
+sources = ['%(version)s.tar.gz']
+
+# gmake posix      -- IOR with only POSIX interface
+# gmake mpiio      -- IOR with only POSIX and MPIIO interfaces
+# gmake hdf5       -- IOR with POSIX, MPIIO, and HDF5 interfaces
+# gmake ncmpi      -- IOR with POSIX, MPIIO, and NCMPI interfaces
+# gmake all        -- IOR with POSIX, MPIIO, HDF5, and NCMPI interfaces
```
Comment (boegel) [inline easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a-mpiio.eb]: @RvDijk comments should updated to mention `bootstrap` instead of `gmake`?

Comment (RvDijk) [inline easybuild/easyconfigs/i/IOR/IOR-3.0.1-foss-2016a-mpiio.eb]: @boegel Updated! In this version, mpiio is default, should it therefore be removed from the easyconfig name?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9fb600d0c2753f9d8ca13fe5b564b54f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2402.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a65d11a7a4edafc69438ecaf5ae2fa94 for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @RvDijk!

**Pull request merged**